crates/generated_parser/src/early_error_checker.rs

```diff
@@ -113,7 +86,9 @@ pub trait EarlyErrorChecker<'alloc> {
         false
     }
 
-    // Check bindings in Script.
+    // Check bindings in Script. This is called after we have identified that
+    // we are in a script. Any remaining bindings should be legal in this context.
```

Comment:
can you mention that we're at the end of the script, that means we have noted all bindings and labels?